<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GSWidget</class>
 <widget class="QWidget" name="GSWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>290</width>
    <height>280</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>40</verstretch>
   </sizepolicy>
  </property>
  <property name="mouseTracking">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout1" rowstretch="0,0,0,0,0,0">
   <item row="0" column="3">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="5" column="1" colspan="5">
    <widget class="QSlider" name="horizontalSlider_exposure">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="5">
    <widget class="QSlider" name="horizontalSlider_ios">
     <property name="palette">
      <palette>
       <active>
        <colorrole role="Base">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>252</red>
           <green>255</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Window">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>236</red>
           <green>128</green>
           <blue>37</blue>
          </color>
         </brush>
        </colorrole>
       </active>
       <inactive>
        <colorrole role="Base">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>252</red>
           <green>255</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Window">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>236</red>
           <green>128</green>
           <blue>37</blue>
          </color>
         </brush>
        </colorrole>
       </inactive>
       <disabled>
        <colorrole role="Base">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>236</red>
           <green>128</green>
           <blue>37</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Window">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>236</red>
           <green>128</green>
           <blue>37</blue>
          </color>
         </brush>
        </colorrole>
       </disabled>
      </palette>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="3" column="1" colspan="5">
    <widget class="QSlider" name="horizontalSlider_iris">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_exposure">
     <property name="text">
      <string>exposure:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_iris">
     <property name="text">
      <string>          iris:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="5">
    <widget class="QLabel" name="fps">
     <property name="text">
      <string>60fps</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_ios">
     <property name="text">
      <string>          iso:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="cam">
     <property name="text">
      <string>Cam1</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QLabel" name="signal_statu2">
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="6">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLabel" name="signal_statu1">
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_color">
     <property name="text">
      <string>       color:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1" colspan="5">
    <widget class="QSlider" name="horizontalSlider_color">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="4">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
